tests/unit_tests/test_spectrum_utils.py

```
@@ -51,15 +52,50 @@ def test_nlq_plq0_squared_matches_known_l2_q0_value():
     assert Nlq_Plq0_squared(2, 0) == pytest.approx(5 / (16 * math.pi))
 
 
+@pytest.mark.parametrize(
+    ("l", "q"),
+    [(2, 0), (2, 2), (3, 1), (6, 4), (10, 3)],
+)
+def test_nlq_plq0_squared_matches_direct_low_mode_calculation(l, q):
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)